valkeyotel/trace.go

```diff
@@ -163,7 +165,7 @@ type otelclient struct {
 	meter           metric.Meter
 	cscMiss         metric.Int64Counter
 	cscHits         metric.Int64Counter
-	tAttrs          trace.SpanStartEventOption
+	tAttrs          []trace.SpanStartOption
```

Comment:
_tbh this should probably be renamed to `spanOpts`, but renaming feels out of scope for this PR, so leaving it for a follow up if/when this PR is accepted._